main.css

```diff
@@ -602,3 +604,138 @@ ul {
   width: 10px;
   font-weight: 600;
 }
+
+.myself {
+  background-size: cover;
+  width: 100%;
+  height: 1263px;
+  display: flex;
+  flex-direction: column;
+  padding: 24px;
+  background-color: #fff;
+  margin-top: 114px;
+  border-top-right-radius: 80px;
+}
+
+.contact__container_me h2 {
+  text-align: center;
+  margin-bottom: var(--bottom-margin-2);
+}
+
+.contact__container_me {
+  position: absolute;
+  width: cover;
+  left: 1px;
+  background: #6070ff;
+  border-top-left-radius: 80px;
+}
+
+.contact__name {
+  position: absolute;
+  background: #fff;
+}
+
+.Contact__me {
+  display: flex;
+  min-width: 327px;
+  height: 56px;
+  flex-direction: column;
+  justify-content: center;
+  flex-shrink: 0;
+  color: #fff;
+  text-align: center;
+  font-family: var(--FF-primary-poppins);
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 52px;
+}
+
+.Contact__text {
+  min-width: 327px;
+  height: 137px;
+  flex-shrink: 0;
+  color: #ebebff;
+  text-align: center;
+  font-family: var(--FF-primary-poppins);
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px;
+  padding-top: 12px;
+  padding-left: 24px;
+  padding-bottom: 12px;
+  padding-right: 24px;
+}
+
+input {
+  display: block;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  background: #fff;
+  min-width: 327px;
+  height: 48px;
+  margin-top: 20px;
+  padding-left: 24px;
+}
+
+textarea {
+  display: block;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  background: #fff;
+  margin-top: 18px;
+  min-width: 327px;
+  height: 170px;
+  padding-left: 24px;
+}
+
+.contact__btn {
```

Comment:
- [ ] You are doing a great job, but you still need to follow the previous reviewer’s feedback and design your button according to the figma template. The issue with your button is that you are using CSS height instead of padding to match the Figma design. Please use padding to adjust the button size and shape. See example below:

.contact__btn {
  display: inline-flex;
  align-items: center;
  border-radius: 8px;
  background: #fff;
  ~height: 42px;~ padding: 5px 10px;
  width: 142px;
  gap: 45px;
}


| Before  | After using padding |
| ------------- | ------------- |
| <img width="62" alt="image" src="https://github.com/AsciencioAlex/Mobile-Finish-Portfolio/assets/52934128/7401289a-5ef1-4e42-8ead-945bbb453634">  | <img width="64" alt="image" src="https://github.com/AsciencioAlex/Mobile-Finish-Portfolio/assets/52934128/06e80013-41b1-4a8d-8edc-3190490a0336">  |
| Content Cell  | Content Cell  |